### DISCUSSION TITLE
Missing intermediate certificate

### DISCUSSION THREAD

**Discussion by plancked** [postalserver/postal#3214] [Development discussions]
Hi,

It seems that TLS works, but partially. I'm getting this report:

  |   | Cert VALIDATION ERROR(S): unable to get local issuer certificate
  |   | This may help: What Is An Intermediate Certificate
  |   | So email is encrypted but the recipient domain is not verified

It's missing the bundle certificate?
My certificate comprises of:

smtp.cert
smtp.key
smtp_bundle.cert

How do I make the bundle file to be used?

Thank you
Alex

**Comment by willpower232** [comment]
I think you just need to combine the certificate and the bundle into a single file.

**Reply by plancked** [reply]
yes, that fixed it.
